automation/taskcluster/androidTest/flank-x86.yml

```diff
@@ -21,6 +21,10 @@ gcloud:
   # see: https://github.com/TestArmada/flank/issues/341
   #results-history-name: tmp_parallel 
 
+  # The number of times a test execution should be re-attempted if one or more failures occur.
+  # The maximum number of reruns allowed is 10. Default is 0, which implies no reruns.
+  num-flaky-test-attempts: 2
```

Comment:
let's do just 1 per our conversation.  My understanding is that 0 will run once and 1 will repeat once